conf/application.conf

```diff
@@ -96,7 +96,7 @@ callbackValidation.allowedProtocols = "https"
 global.file.size.limit = 104857600
 
 auditing {
-  enabled = true
+  enabled = false
```

Comment:
Should auditing be enabled?

Reply:
Auditing is enabled for real environments via app-config-base see: https://github.com/hmrc/app-config-base/blob/8abf8c3754008d7c0be7ac646439ae5a9b2dbf38/upscan-initiate.conf#L9

This just disables it for local development to stop spamming the log with connection failures when a local datastream is not running.